service/src/app/load/pure/persons.py

```
@@ -1,35 +1,232 @@
+            ).all()
+        )
+        found_requested_student_unit_ids = set(
+            session.scalars(
+                select(OrganisationalUnit.id).where(
+                    OrganisationalUnit.id.in_(requested_student_unit_ids)
+                )
+            ).all()
+        )
+
+        if logger is not None:
+            for id in requested_staff_unit_ids.difference(
```

Ruff (0.15.11)

[error] Variable `id` is shadowing a Python builtin

(A001)